Index: src/Command/SystemInstallCommand.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- a/src/Command/SystemInstallCommand.php	(revision 951d9097859e2a5d57529ac2223ea88894ed1e77)
+++ b/src/Command/SystemInstallCommand.php	(date 1603316621590)
@@ -48,14 +48,6 @@
     {
         $output = new ShopwareStyle($input, $output);
 
-        /**
-         * Needs to be set because migration for testsuite needs the trigger.
-         * Because there are some tests that work directly on the db and so ignore the indexer
-         */
-        putenv('BLUE_GREEN_DEPLOYMENT=1');
-
-        $_ENV['BLUE_GREEN_DEPLOYMENT'] = 1;
-
         $dsn = trim((string)($_ENV['DATABASE_URL'] ?? $_SERVER['DATABASE_URL'] ?? getenv('DATABASE_URL')));
         if ($dsn === '' || $dsn === Kernel::PLACEHOLDER_DATABASE_URL)  {
             $output->error("Environment variable 'DATABASE_URL' not defined.");
